### DISCUSSION TITLE
text/textarea make larger

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#375] [Q&A] [answered]
If i use a text or a textarea for a User to give them a possability to write more text.
How can i entere a custom field with multipleLines bigger text?

**Comment by liedekef** [comment]
I'm not exactly sure, but if you talk about larger font sizes you should look into CSS for your theme. EME obeys the CSS rules set by your theme (or anything else).

**Reply by loelismurf** [reply]
I talk about the size of the box not the font
![image](https://github.com/liedekef/events-made-easy/assets/77551562/90f3ed68-6b52-46fe-81b9-73883e85148f)
So that are more space for more

**Reply by liedekef** [reply, marked as answer]
In that case: when defining a custom field, you have an extra setting 'HTML field attributes' where you can define the height/width/... of your field. In your case you'll need to set "cols" and "rows" in that option.
